snap-patch/dnsmasq.patch

```diff
@@ -96,7 +96,6 @@ index ce44809..17c451b 100644
 +	  /*   }      */
  
  #ifdef HAVE_LINUX_NETWORK
- 	  if (is_dad_listeners() || option_bool(OPT_CLEVERBIND))
```

Comment:
why is this removed? is this change now in the sources?

Reply:
We're only reducing the context of the patch here by one line. As those surroundings changed in the ESM version. We keep the actual logical changes, though.

The `setuid`/`setgid` calls are still there in the ESM version and will continue to be commented out by the patch.